### ISSUE TITLE
When NVDA is on and user navigates on Learn more link with the help of down arrow key, Nvda does not announces its control type

### ISSUE THREAD

**Issue by Yash14j** [microsoft/vscode#128048]
**GitHubTags:**#A11y_VSCode;#A11ySev2;#A11yTCS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;#A11yMAS;#FTP;#STP;#JAWS;#NVDA;
### Actual:
When NVDA is on and user navigates on Learn more link with the help of down arrow key, Nvda does not announces its control type

**Similar issue is observed with JAWS**

**Similar issue is observed with Kubernetes Bikes accessibility Testing cluster listitems.**

### Expected:

When NVDA is on and user navigates on Learn more link with the help of down arrow key, Nvda should announce it as "result 3/3 learn more link"

### Environment Details:
Application Name: Bridge to Kubernetes (Mindaro)
Windows Version: Windows10
### Repro Steps:

1. Setup The Kubernetes extension.
2. Open visual studio code and load "Bikes" project.
3. Tab to "View" menu item in top navigation bar, press enter to expand and use down arrow key to Select "Command Pallet" option.
4. Search and select "Local Process with Kubernetes: Configure Local Process with Kubernetes”
5. Connect to Kubernetes panel will open.
6. Tab to bike options and press enter.
7. Input “3000” as your local port and press enter.
8. Tab to Launch via NPM option and press enter to select it.
9. Open NVDA and navigate on learn more link with the help of right arrow key and check the result

### User Impact:
Screenreader user will not be able to know about the control type of Learn more

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
